tools/post-merge

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+echo ""
+echo ""
+echo ""
+echo "#########################################################################"
+echo "#                                                                       #"
+echo "# !!!WARNING!!!                                                         #"
+echo "#                                                                       #"
+echo "# Must execute:                                                         #"
+echo "#                                                                       #"
+echo "#    \`env -i bash\`                                                    #"
```

Comment:
s/bash/$(basename "$SHELL")/ so it stays accurate for zsh as well :)

Reply:
Can do :)